zdm/data/Surveys/CRAFT_ICS_892.dat

```diff
@@ -0,0 +1,15 @@
+BW    336 #MHz
```

Comment:
Just a reminder that this Repo is public.
So be mindful of what shows up in files like these.

Reply:
yes, fraid so.